### PR TITLE
Takes into account all executions for more accuracy

### PR DESCRIPTION
**Pull request by quanturium**
This adds a little more accuracy to the data but not really any changes in google finance. My guess is google finance does not use the timestamp of the transaction really precisely. 

The second thing that I could not find a way to fix is when a company offers you stocks. In this case it appears in your robinhood portfolio but is neither buy or sell operation so there is no transaction associated to it (as far as I could find on the transaction API)

### CODE REVIEW

File: gf-export.py [hunk 1]
```diff
@@ -61,19 +61,23 @@
...
-            #      fields[i+(page*100)][value] = executions[0][value]
+            for execution in executions: # Iterate over all the different executions
+                fields[n]['symbol'] = instrument['symbol']
+                for key, value in enumerate(order):
+                    if value != "executions":
+                        fields[n][value] = order[value]
+                fields[n]['timestamp'] = execution['timestamp']
+                fields[n]['quantity'] = execution['quantity']
+                fields[n]['price'] = execution['price']
+                n+=1
```
Comment (briansrls) [inline gf-export.py:86]: I think making this block here is a good idea, but I would like to make sure that moving this under 

```
if order['state'] == "filled":
              trade_count += 1
```
 
doesn't create redundancies later on.  

For example, if we check if the state is filled here, there is no reason to check later on which is nice.  
  
```
 if str(fields[row]["state"]) == "filled" and str(fields[row]["cancel"]) == "None":
              try:
                  if key!="state" and key!="cancel":
                     if key=="average_price" or key=="fees":
                         fields[row][key] = round(float(fields[row][key]),2)
                      line += str(fields[row][key]) + ","
              except:
                  line += ","
```

Here we should be able to remove the opening if statement checking of the state is filled. 

If we don't need to check if this if statement is true (checking if cancel is "none" may also be redundant, I was never sure what that column was for?), then we may not have to add "state" and "cancel" to the desired keys, and then to the new keys list.  Let me know if what I'm saying makes sense.

All in all, your code is certainly an improvement over what exists.  I think you can just remove the if statement I mentioned in the second block here, and check if there is any need to even have state/cancel fields in our desired list.  Test and let me know if that still works, and I'll merge everything.

File: gf-export.py [hunk 2]
```diff
@@ -99,7 +103,7 @@
 
 # CSV headers
 
-desired = ("average_price", "created_at", "fees", "quantity", "symbol", "side", "state", "cancel")
+desired = ("price", "timestamp", "fees", "quantity", "symbol", "side", "state", "cancel")
```
Comment (briansrls) [inline gf-export.py]: Can we remove state, cancel as well?

**Review by briansrls — CHANGES_REQUESTED** [review]

File: gf-export.py (hunk 1 above)
Comment (quanturium) [reply gf-export.py:86]: Fixed what you've mentioned in latest comments

**Review by briansrls — APPROVED** [review]
Excellent, thanks a lot for the contribution.  I think overall this is a much better way of doing what I wanted in the first place.

**Pull request merged**